cmake/common.cmake

```diff
@@ -184,16 +187,23 @@ if ("${TARGET_OS}" STREQUAL "QNX")
          tiipc-usr
          ti-udmalld
          ti-pdk
+         ti-csirxlld
```

Comment:
csirx, fvid2 libraries are QNX specific for AM62A only, for Jacinto these are on R5. Please add AM62A check here